docs/reference/environment-variables.rst

```diff
@@ -8,6 +8,35 @@
 Environment variables
 *********************
 
+Environment variables can be used to influence ``meson-python``'s behavior, as
+well as the behavior of Meson and other tools that may be used during the
+build. This page lists all the environment variables directly used by
+``meson-python``.
+
+Meson recommends using command line arguments instead of environment variables,
+but does support a number of environment variables for compatibility with other
+build systems:
+
+- `Compiler and linker flag variables <https://mesonbuild.com/Reference-tables.html#compiler-and-linker-flag-environment-variables>`__
+- `Compiler and linker selection variables <https://mesonbuild.com/Reference-tables.html#compiler-and-linker-selection-variables>`__
+
+Other tools each may support their own variables. E.g., ``pkg-config`` supports
+``PKG_CONFIG_PATH`` which influences the search path for dependencies at build
+time.
```

Comment:
Technically `pkg-config` is used at setup (configure) time, not at build time. Maybe "at build time" can be simply omitted without loss of relevant information. Also, maybe, "the search path for dependencies" should be something like "the search path for ``.pc`` files describing the available dependecies".

Reply:
agreed, adopted